gradle-plugins/build.gradle.kts

```diff
@@ -72,4 +82,15 @@ tasks {
   publishPlugins {
     enabled = !version.toString().contains("SNAPSHOT")
   }
-}
\ No newline at end of file
+
+  val extractBbGradlePlugin by registering(Copy::class) {
+    from(zipTree(bbGradlePlugin.files.first { it.name.contains("byte-buddy-gradle-plugin") })) {
```

Comment:
Honestly, this looks like a hack. Why exactly do we need to unzip a dependency jar file into our `classes` folder instead of putting jar file to a classpath?